ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -836,6 +854,7 @@ static bool TryGetVirtualizationRoot(
         return false;
     }
     
+    *vnodeFsidInode = Vnode_GetFsidAndInode(vnode, context, true /* the inode is used for getting the path in the provider, so use linkid */);
```

Comment:
did we move this to the end just to be consistent or was the move needed?
Also if i was being super nit picky I'd say in a function below you have a newline between vnodeFsidInode and return ;-).  Not something that needs to change.

Reply:
> did we move this to the end just to be consistent or was the move needed?

I moved it because if `TryGetVirtualizationRoot` returns `false` there's no point in looking up the vnode`s fsid and inode.